ocean_lib/models/dtfactory.py

```diff
@@ -14,50 +13,36 @@ class DTFactory(ContractBase):
     FIRST_BLOB = "https://example.com/dataset-1"
 
     def verify_data_token(self, dt_address):
+        """Checks that a token was registered."""
         event = getattr(self.events, "TokenRegistered")
         filter_params = {"tokenAddress": dt_address}
         event_filter = event().createFilter(fromBlock=0, argument_filters=filter_params)
         logs = event_filter.get_all_entries()
 
         return logs and logs[0].args.tokenAddress == dt_address
 
-    def get_token_registered_event(
-        self, from_block, to_block, metadata_url=None, sender=None, token_address=None
-    ):
+    def get_token_registered_event(self, from_block, to_block, token_address):
+        """Retrieves event log of token registration."""
         event = getattr(self.events, "TokenRegistered")
-        filter_params = {}
-        if token_address:
-            filter_params["tokenAddress"] = token_address
+        filter_params = {"tokenAddress": token_address}
 
         event_filter = event().createFilter(
             fromBlock=from_block, toBlock=to_block, argument_filters=filter_params
         )
         logs = event_filter.get_all_entries()
-        if logs and token_address:
-            return logs[0]
 
-        for log in logs:
-            if log.args.blob == metadata_url and sender == log.args.RegisteredBy:
-                return log
-
-        return None
+        return logs[0] if logs else None
 
     def get_token_minter(self, token_address):
-        event = getattr(self.events, "TokenRegistered")
-        # TODO: use the filter on tokenAddress when it is set as indexed in the contract event.
-        filter_params = {}  # {'tokenAddress': token_address}
-        event_filter = EventFilter(
-            "TokenRegistered", event, filter_params, from_block=0, to_block="latest"
-        )
-        logs = event_filter.get_all_entries(max_tries=10)
-        for log in logs:
-            # assert log.args.tokenAddress == token_address
-            if log.args.tokenAddress == token_address:
-                return log.args.registeredBy
+        """Retrieves token minter.
+
+        This function is deprecated and only kept for maintainability purposes."""
```

Comment:
Suggest replace "This function .. purposes" with 
"This function will be deprecated in the next major release. It's only kept for backwards compatibility".